Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Db.ts

```diff
@@ -63,6 +63,11 @@ export async function createConnectionPoolAndReturnClient(): Promise<PoolClient>
     // database, and disconnect. From there reconnect the pool to the meadowlark database and continue
 
     singletonDbPool = new Pool(dbConfiguration);
+
+    singletonDbPool.on('error', (err, _client) => {
+      Logger.error(`${moduleName} a PostgreSQL connection error occurred.`, null, err);
+    });
+
```

Comment:
I didn't realize I had committed this. Came in while I was exploring this problem with Andony earlier in the week. Wonder if this is actually a better  place to put this, before the `connect` command instead of after it? Let's see what happens when the e2e tests run in GitHub.